src/momblish/momblish.py

```diff
@@ -45,11 +46,20 @@ def word(self, length=None):
 
         for _ in range(length-2):
             last_bigram = word[-2:]
-            next_letter = random.choices(
-                    list(self.corpus.occurences[last_bigram]),
-                    weights=self.corpus.occurences[last_bigram].values())[0]
+            try:
```

Comment:
it's possible to test this by using a corpus with one bad word and one good word in it.  run it a couple of dozen times and verify that it only ever generates the good word.